CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,57 @@
+  - Obscene or intimidating gestures
+  - Shouting
+  - Sustained disruption of talks and events
+  - Disruption of meetings
+  - Inappropriate physical contact
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying information ("doxing")
+
+
+- No matter who you are, if you feel you have been or are being harassed or made uncomfortable by another person in relation to your work on FPF projects, please report it immediately.  We are aware that harassment and sexual violence are systemic and pervasive social issues that demand continued awareness and attention, and that a code of conduct is only a small component of that.  We are committed to taking proactive action to make our organizational spaces safe and comfortable for everyone.
+
+
+- **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Exclusionary behavior is absolutely unacceptable. This includes, but is not limited to:
+  - Discriminatory jokes and language.
+  - Posting sexually explicit or violent material.
+  - Personal insults, especially those using racist or sexist terms.
+  - Unwelcome sexual attention.
+  - Advocating for, or encouraging, any of the above behavior.
+
+
+- **When we disagree, try to understand why.** Disagreements — both social and technical — happen all the time, even at Freedom of the Press Foundation. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of FPF comes from its varied team, people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn't mean that they’re wrong. Don’t forget that it is human to err and blaming each other doesn't get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+
+- **Name things appropriately.** Avoid giving your user accounts, servers, repositories, etc. names that are overtly sexual or that might otherwise detract from a friendly, safe and welcoming environment for all.
+
+## Sanctions
+
+We will deal with violations of this code of conduct on a case by case basis, depending on the severity of the behavior and any prior violations. Sanctions may range from a simple warning not to engage in a particular kind of behavior (e.g., in response to a single mildly inappropriate comment), to separation from FPF (termination in the case of employment, removal from all FPF technical and social spaces in the case of volunteers) and other consequences in the most severe cases.
+
+Original text courtesy of the [Django project](https://www.djangoproject.com/conduct/), which in turn was inspired by the [Speak Up! Code of Conduc](https://web.archive.org/web/20141109123859/http://speakup.io/coc.html)t. License: [Creative Commons Attribution](https://creativecommons.org/licenses/by/4.0/) (CC-BY).
```

Comment:
Can we make this CC-BY-SA? i would slightly prefer that.

Reply:
Since it's based on other people's work, I would prefer to respect their original licensing choice unless there are compelling reasons to change it.

Reply:
Fair enough

Reply:
Agreed with @eloquence on that point.